players/installing-java/linux.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Installing Java on Linux
+description: A step by step guide on how to install Java on Linux.
+---
+
+# Installing Java on Linux
+
+This guide will walk you through installing Java 17 on Linux.
+
+The Minecraft Launcher comes with its own java installation, so this section is only relevant if you want to use the Fabric `.jar` based installer, or if you want to use the Minecraft Server `.jar`.
+
+## 1. Verify if Java is already installed.
+
+Open a terminal and type `java -version` and press <kbd>Enter</kbd>.
+
+If the command runs successfully, you will see something like this. If the command failed, proceed to the next step.
+
+![Terminal with "java -version" typed in.](/assets/players/installing-java/linux-java-version.png)
+
+::: warning
+To use the majority of modern Minecraft versions, you'll need at least Java 17 installed. If this command displays any version lower than 17, you'll need to update your existing java installation.
+:::
+
+## 2. Downloading and Installing Java 17
```

Comment:
We should recommend JDK for all users.

Reply:
Agreed, the official launcher ships JDK, so that should be expected.